Sorting/DateSorting.php

```diff
@@ -29,10 +29,10 @@ class DateSorting extends AbstractOrderSorting
      */
     protected function compare(CommentInterface $a, CommentInterface $b)
     {
-        if ($a->getCreatedAt() === $b->getCreatedAt()) {
+        if ($a->getCreatedAt()->getTimestamp() === $b->getCreatedAt()->getTimestamp()) {
             return 0;
         }
 
-        return $a->getCreatedAt() < $b->getCreatedAt() ? -1 : 1;
+        return $a->getCreatedAt()->getTimestamp() < $b->getCreatedAt()->getTimestamp() ? -1 : 1;
```

Comment:
this is not needed here. This comparison operator was working fine. PHP is able to compare DateTime objects